### PR TITLE
Allow passing --disable-spring option to test_launcher

### PR DESCRIPTION
**Pull request by darreneng**
This PR enables passing the `--disable-spring` option to [test_launcher](https://github.com/petekinnecom/test_launcher#usage-and-options)

If a user adds this to their `settings.json`
```
    "rubytestlauncher.disablespring": true,
```

then test_launcher will receive the option in the terminal, e.g.

```
test_launcher /path/to/file --disable-spring
```

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -48,7 +48,8 @@ export function activate(context: vscode.ExtensionContext) {
         const terminal = findOrCreateTerminal();
 
         terminal.show();
-        terminal.sendText(`test_launcher ${launcher_args}`);
+        const launcherOptions = vscode.workspace.getConfiguration(undefined, null).get('rubyTestLauncher.disableSpring') ? '--disable-spring' : ''
+        terminal.sendText(`test_launcher ${launcher_args} ${launcherOptions}`);
```
Comment (blakeherrington) [inline src/extension.ts]: Come on @darreneng. Where is the unit test coverage for this masterpiece of an extension?

File: src/extension.ts [hunk 2]
```diff
@@ -48,7 +48,8 @@ export function activate(context: vscode.ExtensionContext) {
         const terminal = findOrCreateTerminal();
 
         terminal.show();
-        terminal.sendText(`test_launcher ${launcher_args}`);
+        const launcherOptions = vscode.workspace.getConfiguration(undefined, null).get('rubyTestLauncher.disableSpring') ? '--disable-spring' : ''
```
Comment (blakeherrington) [inline src/extension.ts]: From the docs, it looks like the second argument to `getConfiguration` defaults to null, so we can probably omit it.

File: src/extension.ts (hunk 1 above)
Comment (blakeherrington) [reply src/extension.ts]: lol if you're feeling generous, go ahead and correct `launcher_args` to `launcherArgs`.

The idea of args vs. options here is really not clear (and in it's usage above), but I suppose basically the options are what comes from configuration at this point - I think it's fine as is. 🤷‍♂️

Comment (darreneng) [reply src/extension.ts]: tests are for losers :wink:

yeah I agree... my line of thought is one comes from the configuration and the other is a command, but the fact that one of the commands is triggered via `--rerun` makes this pretty confusing.

**Review by blakeherrington — APPROVED** [review]

**Pull request merged**